tests/benchmark/run_parallel.py

```diff
@@ -89,9 +89,9 @@ class Scenario:
 	ops: int = 10
 
 	def print(self):
-		res = f"✨ Scenario: {'🔹' * self.readers}{'🔻' * self.writers} ({self.readers}r{self.writers}w)"
-		res += ", 🔸 compression" if self.use_compression else ""
-		res += ", 💎 big file" if self.big_file else ""
```

Comment:
Was this actually an issue? I think about every terminal and editor can handle unicode by now

Reply:
in my pycharm terminal this is not displayed correctly

Reply:
![image](https://user-images.githubusercontent.com/20727911/203658179-df7c6daa-217d-403b-95a7-d144de710f69.png)

Reply:
Damn I already thought my ssd is fast, but you get more than twice the ops/sec as me!